upcoming/gopher-gala.md

```diff
@@ -0,0 +1,23 @@
++++
+author = ["Casey Rosengren"]
+date = "2015-1-3"
+title = "The world's first Golang distributed hackathon."
```

Comment:
Go~~lang~~. likewise below. It's a tag but the name of the language is just "Go"

javascriptlang, rubylang, scalalang. you get the idea.